ansible/roles/k8s-master/tasks/main.yml

```diff
@@ -1,4 +1,15 @@
 ---
+- name: determine if feature gates file exists
+  delegate_to: localhost
+  stat:
+    path: feature_gates_{{ k8s_version }}.yml
```

Comment:
The feature gate file is named by the full X.Y.Z of the kube version, but the needed gates only change w/ X.Y.

I think a `set_fact` that has a `regex_replace` could be used to get the major/minor from `k8s_version` so that the feature gate file is 1-per minor kube version.